cri-o/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM registry.fedoraproject.org/fedora:rawhide
+
+ENV VERSION=0 RELEASE=1 ARCH=x86_64
+LABEL com.redhat.component="cri-o" \
+      name="$FGC/cri-o" \
+      version="$VERSION" \
+      release="$RELEASE.$DISTTAG" \
+      architecture="$ARCH" \
+      usage="atomic install --system --system-package=no crio && systemctl start crio" \
+      summary="The cri-o daemon as a system container." \
+      maintainer="Yu Qi Zhang <jzehrarnyg@gmail.com>" \
+      atomic.type="system"
+
+COPY tmpfiles.template config.json.template service.template /exports/
+
+RUN dnf install --setopt=tsflags=nodocs -y cri-o cri-o-cni && \
+    dnf clean all && \
+    mkdir -p /exports/hostfs/etc/crio && \
+    cp /etc/crio/* /exports/hostfs/etc/crio
```

Comment:
Since we are bind mounting the entire ``/etc/`` into the container should ``/etc/containers/policy.json`` also be copied to the host? Otherwise, we need to ensure that ``skopeo-containers`` is installed or something else handles that file.

Ref: https://github.com/kubernetes-incubator/cri-o/blob/master/kubernetes.md#preparing-crio